src/jesse_state.erl

```diff
@@ -61,7 +62,7 @@
                                      ) -> list() | no_return()
                                             )
          , default_schema_ver :: binary()
-         , schema_loader_fun  :: fun(( binary()
+         , schema_loader_fun  :: fun(( string()
```

Comment:
correct me if I'm wrong, but this is just a spec correction, because even before this PR, the `Key` (argument to schema_loader_fun) was a string, since that is what `get_schema_id/1` was returning.

You can even see that before this PR, we were converting binary to string before calling `schema_loader_fun` https://github.com/for-GET/jesse/pull/24/files#diff-70cc40937373cde9cea9266b655884c0L292